e2e/registration.spec.ts

```
@@ -1,124 +1,139 @@
+            email,
+            password,
+            phoneCode,
+            countryCode,
+            phone,
+            linkedin,
+          },
+        ),
+        pageManager.atKeycloakPage.getAdminToken({
+          project,
+          apiContext,
+          adminName: process.env.ADMIN_KEYCLOAK!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)